lib/Imageview.dart

```diff
@@ -107,130 +110,125 @@ class _ImageviewState extends State<Imageview> {
                   files[index],
                 ),
               ),
-              Expanded(
-                flex: (MediaQuery
-                    .of(context)
-                    .size
-                    .height / 18).floor(),
-                child: Container(
-                  height: 65,
-                  color: themeChange.darkTheme ? Colors.black87 : Colors.blue[600],
-                  child: Padding(
-                    padding: const EdgeInsets.all(10.0),
-                    child: Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                      children: <Widget>[
-                        FlatButton(
-                          onPressed: () {
-                            Navigator.of(context).pop();
+            ),
+            Expanded(
+              flex: (MediaQuery.of(context).size.height / 18).floor(),
+              child: Container(
+                height: 65,
+                color:
+                    themeChange.darkTheme ? Colors.black87 : Colors.blue[600],
+                child: Padding(
+                  padding: const EdgeInsets.all(10.0),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: <Widget>[
+                      TextButton(
+                        onPressed: () {
+                          Navigator.of(context).pop();
 //                                  Navigator.of(context).pushReplacement(MaterialPageRoute(
 //                                      builder: (context) => Home()));
-                          },
-                          child: Column(
-                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                            children: <Widget>[
-                              Icon(
-                                Icons.arrow_back,
-                                color: Colors.white,
-                              ),
-                              Text(
-                                "Back",
-                                style: TextStyle(color: Colors.white),
-                              ),
-                            ],
-                          ),
+                        },
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: <Widget>[
+                            const Icon(
```

Comment:
remove `const` and remove it where ever it has been repeated.

Reply:
@arteevraina I added this because of the lint rules. Is it wrong ?

Reply:
No, it is not wrong. But I think it can do the work without that too. Can you tell me which rules are you actually referring?

Reply:
### Please read the last line in that box Sir

**"Prefer const with constant constructors."**

![2021 03 15-22 15](https://user-images.githubusercontent.com/29424616/111189596-41501a00-85dc-11eb-923b-efc87a2312d2.png)